### PR TITLE
Allow to create new team

### PR DESCRIPTION
**Pull request by muhtimur**
Signed-off-by: muhtimur <timur.mukhamedishin@xored.com>

# Contribution checklist

## Brief description

Now we can create teams

image

## Checklist

* [ ] - UI test added to added/changed functionality?
* [x] - Screenshot is added to PR if applicable ?
* [x] - Does the code work? Check whether function and logic are correct.
* [ ] - Does Changelog.md is updated with changes?
* [x] - Does the translations are up to date?
* [x] - Does it well tested?
* [x] - Tested for Chrome.
* [x] - Tested for Safari.
* [x] - Go through the changed code looking for typos, TODOs, commented LOCs, debugging pieces of code, etc.
* [x] - Rebase your branch onto master and upstream branch
* [x] - Is there any redundant or duplicate code?
* [x] - Are required links are linked to PR?
* [ ] - Does new code is well documented ?

## Related issues

[TSK-108](https://front.hc.engineering/workbench/platform/tracker/my-issues#tracker%3Acomponent%3AEditIssue%7C62a8296e4b1f5cf4dffd018e%7Ctracker%3Aclass%3AIssue%7Ccontent)

### CODE REVIEW

File: packages/core/src/classes.ts [hunk 1]
```diff
@@ -271,6 +271,7 @@ export interface Space extends Doc {
   private: boolean
   members: Arr<Ref<Account>>
   archived: boolean
+  icon?: Asset
```
Comment (haiodo) [inline packages/core/src/classes.ts]: We do not need it for generic spaces.

Comment (muhtimur) [reply packages/core/src/classes.ts]: We discussed it with Denis, and he told me that the easiest way to add a custom icon is to create this field in Space interface. But there is one more way -- to create a mixin for Space which will show some presenter with the custom icon. So, is it better to use 2nd way?

Comment (haiodo) [reply packages/core/src/classes.ts]: Is it only will be used for Team? If so please use it there, and have options for Space unified components. No need for mixin here.

Comment (haiodo) [reply packages/core/src/classes.ts]: We should not mix Core and UI in one.

Comment (muhtimur) [reply packages/core/src/classes.ts]: Now it is used only for teams but in future we will have to use it for userpics in direct messages

Comment (muhtimur) [reply packages/core/src/classes.ts]: Added a presenter component for Team

File: packages/core/src/classes.ts [hunk 2]
```diff
@@ -271,6 +271,7 @@ export interface Space extends Doc {
   private: boolean
   members: Arr<Ref<Account>>
   archived: boolean
+  presenter?: AnyAttribute
```
Comment (haiodo) [inline packages/core/src/classes.ts]: Why we need to extend base Space class?

Comment (haiodo) [reply packages/core/src/classes.ts]: Please use some of view.mixin.* if you need presenters.

Comment (muhtimur) [reply packages/core/src/classes.ts]: Corrected it

**Review by haiodo — APPROVED** [review]

**Pull request merged**